ui/src/components/Header/Controller.tsx

```diff
@@ -79,10 +87,15 @@ export const Controller = (): ReactElement => {
 
   const start = async () => {
     const images = await ddClient.docker.listImages() as [DockerImage];
-    const haveLocally = images.some(image => image.RepoTags?.at(0) === LATEST_IMAGE);
+
+    const isPro = runConfig.find(config => config.name === runningConfig)
```

Comment:
nit: unrelated to this PR, but maybe `runConfig` could be renamed to `runConfigs`? (seems to be a list of configs). Not critical though, could be done in a future iteration..